common/utils/src/main/java/org/apache/spark/internal/Logger.java

```diff
@@ -34,10 +34,26 @@ public class Logger {
     this.slf4jLogger = slf4jLogger;
   }
 
+  public boolean isErrorEnabled() {
+    return slf4jLogger.isErrorEnabled();
+  }
+
   public void error(String msg) {
     slf4jLogger.error(msg);
   }
 
+  public void error(String msg, String arg) {
+    slf4jLogger.error(msg, arg);
+  }
+
+  public void error(String msg, String arg1, String arg2) {
+    slf4jLogger.error(msg, arg1, arg2);
+  }
+
+  public void error(String msg, String... args) {
```

Comment:
Why is the method signature not `public void error(String msg, Object... args)`
Because it conflicts with `public void error(String msg, MDC... mdcs)`
When we want to use `error(String msg, MDC... mdcs)`, it will enter `error(String msg, Object... args)`